licytacja/konwencje-strefy-szlemowej/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Konwencje strefy szlemowej
+layout: default
+---
+
+# Konwencje strefy szlemowej
+
+Konwencje sterfy szlemowej pozwalają uzyskać dokładne informację przy grze szlemowej czy szlemikowej.
```

Comment:
Ja bym to zmienił na stwierdzenie, że pozwalają ustalić, czy w połączonych rękach jest bilans na grę w szlemika lub szlema.

Reply:
poprawione w najnowyszm commitcie